var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -40,6 +41,9 @@ class Charliecloud(MakefilePackage):
     depends_on('py-sphinx',           type='build', when='+docs')
     depends_on('py-sphinx-rtd-theme', type='build', when='+docs')
 
+    # bash automated testing harness (bats)
+    depends_on('bats@0.4.0', type='test')
```

Comment:
This is the only version that works?

Reply:
AFAICT, yes. FWIIW, 0.4.0 is the latest version and was released Auguest 13, 2014. The project is longer being developed.

There is another project (fork) called bats-core but we just use bats 0.4.0.